docker/client/main.tf

```diff
@@ -1,10 +1,17 @@
 variable "ca_cert_pem" {}
 variable "ca_private_key_pem" {}
-variable "ip_addresses_list" {}
 # supports if you have a public/private ip and you want to set the private ip
 # for internal cert but use the public_ip to connect via ssh
-variable "deploy_ssh_hosts" {}
-variable "dns_names_list" { default = "*.*.cluster.internal,*.ec2.internal" }
+variable "deploy_ssh_hosts" {
+    type = "list"
```

Comment:
I thought the only types available were string and map? https://www.terraform.io/docs/configuration/variables.html